pwsh_module/pwsh_bolt_internal.ps1

```diff
@@ -68,3 +68,17 @@ function Get-BoltCommandline {
 
   Write-Output $params
 }
+
+function Get-BoltVersion{
```

Comment:
Does it make sense to export this function so it can be used on its own?

Reply:
I think it makes sense to make it a module export because the only way to get the version currently is to use a `get` cmdlet like `Get-BoltTask -version` and that doesn't make much sense.